### PR TITLE
Rename `workflow_steps.datastream` to `workflow_steps.workflow`

### LINKED ISSUE #43: Rename "datastream" to "workflow_name"

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #43

### CODE REVIEW

File: bin/export-legacy-workflow-archive [hunk 1]
```diff
@@ -12,7 +12,7 @@ require 'ruby-oci8'
 require 'csv'
 
 conn = OCI8.new(ARGV[0], ARGV[1], ARGV[2])
-columns = %w[id druid datastream process status error_msg error_txt datetime attempts
+columns = %w[id druid workflow process status error_msg error_txt datetime attempts
```
Comment (jcoyne) [inline bin/export-legacy-workflow-archive]: This is for the legacy database, we haven't renamed it there.

File: bin/export-legacy-workflow [hunk 2]
```diff
@@ -12,7 +12,7 @@ require 'ruby-oci8'
 require 'csv'
 
 conn = OCI8.new(ARGV[0], ARGV[1], ARGV[2])
-columns = %w[id druid datastream process status error_msg error_txt datetime attempts
+columns = %w[id druid workflow process status error_msg error_txt datetime attempts
```
Comment (jcoyne) [inline bin/export-legacy-workflow]: This is for the legacy database, we haven't renamed it there.

File: bin/export-legacy-workflow-archive (hunk 1 above)
Comment (mjgiarlo) [reply bin/export-legacy-workflow-archive]: OK.

File: bin/export-legacy-workflow (hunk 2 above)
Comment (mjgiarlo) [reply bin/export-legacy-workflow]: Gotcha.

**Comment by mjgiarlo** [comment]
> We also need to update `bin/import-legacy*` to rename the old `datastream` column to `workflow`

Good call. Pushed.

File: app/views/workflows/index.xml.builder [hunk 3]
```diff
@@ -1,10 +1,13 @@
 # frozen_string_literal: true
 
 xml.workflows(objectId: params[:druid]) do
-  @processes.each do |datastream|
-    xml.workflow(repository: params[:repo], objectId: params[:druid], id: datastream[0]) do
-      datastream[1].each do |workflow|
-        workflow.as_process(xml)
+  @workflow_steps.each do |workflow_step|
+    workflow = workflow_step[0]
```
Comment (mjgiarlo) [inline app/views/workflows/index.xml.builder]: While we don't need these variables, I added them to give names to things, thereby clarifying their purpose.

File: app/views/workflows/show.xml.builder [hunk 4]
```diff
@@ -1,9 +1,12 @@
 # frozen_string_literal: true
 
-@processes.each do |datastream|
-  xml.workflow(repository: params[:repo], objectId: params[:druid], id: datastream[0]) do
-    datastream[1].each do |workflow|
-      workflow.as_process(xml)
+@workflow_steps.each do |workflow_step|
+  workflow = workflow_step[0]
```
Comment (mjgiarlo) [inline app/views/workflows/show.xml.builder]: While we don't need these variables, I added them to give names to things, thereby clarifying their purpose.

**Pull request merged**